src/SprykerEco/Zed/Heidelpay/Business/HeidelpayBusinessFactory.php

```diff
@@ -110,6 +148,32 @@ public function createDebitTransactionHandler(): DebitTransactionHandlerInterfac
         );
     }
 
+    /**
+     * @return \SprykerEco\Zed\Heidelpay\Business\Payment\Transaction\Handler\FinalizeTransactionHandlerInterface
+     */
+    public function createFinalizeTransactionHandler()
+    {
+        return new FinalizeTransactionHandler(
```

Comment:
Same. You should not have verb as a class name. There are a few exception, but not in this case